### ISSUE TITLE
[Ubuntu] integrated menu bar dissappears

### ISSUE THREAD

**Issue by kapad** [baumgarr/Nixnote2#397]
In Ubuntu, 

After I minimize nixnote2 to tray and then do a show again, the integrated menu bar disappears. This persists until I close and open nixnote2.

*The status bar remains, but the different options for nixnote2 in the status bar disappear.

**Comment by robert7** [comment]
As you maybe noticed, the original maintainer isn't available now.. It someway happened, that I took over.. There is updated version of the app ("[v.2.1](https://github.com/robert7/nixnote2/releases)") in [my fork](https://github.com/robert7/nixnote2). So far I did mainly stabilization/bugfixing and added few new features. 

I could reproduce this. It seems to be Qt bug in some particular Qt version (~5.5.*)

Try the new AppImage version. Seems to work there fine

**Comment by kapad** [comment]
hehe.. this is almost a year old. i suspect some scripting here. :)

**Comment by robert7** [comment]
yes. I simply gone through all unanswered questions and posted some meaningfull answer where possible. and no I'm not a bot (as far I'm know - you can never be really sure)

**Comment by kapad** [comment]
Hahaha.. well, I'm already on your version and hope this will get others to
move to it too!
